docs/content/guides/author-apps/dapr/how-to-dapr-sidecar/index.md

```diff
@@ -0,0 +1,76 @@
+---
+type: docs
+title: "How-To: Add a Dapr sidecar to Radius"
+linkTitle: "Add a Dapr sidecar to Radius"
+description: "Easily leverage a Dapr sidecar blocks in your application for code and infrastructure portability"
+weight: 100
+categories: "How-To"
+tags: ["Dapr"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Leverage a [Dapr sidecar](https://docs.dapr.io/concepts/dapr-services/sidecar/) with your Radius Application
+
+## Prerequisites
+
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Dapr](https://docs.dapr.io/getting-started/install-dapr-cli/)
```

Comment:
Another prereq should be a supported Kubernetes cluster, along with Dapr initialized with `dapr init -k`